src/shortestpaths/dijkstra.jl

```diff
@@ -20,19 +20,21 @@ Return a [`Graphs.DijkstraState`](@ref) that contains various traversal informat
 
 
 ### Optional Arguments
-* `allpaths=false`: If true, 
+* `allpaths=false`: If true,
 
 `state.predecessors` holds a vector, indexed by vertex,
 of all the predecessors discovered during shortest-path calculations.
 This keeps track of all parents when there are multiple shortest paths available from the source.
 
-`state.pathcounts` holds a vector, indexed by vertex, of the number of shortest paths from the source to that vertex. 
+`state.pathcounts` holds a vector, indexed by vertex, of the number of shortest paths from the source to that vertex.
 The path count of a source vertex is always `1.0`. The path count of an unreached vertex is always `0.0`.
 
-* `trackvertices=false`: If true, 
+* `trackvertices=false`: If true,
 
 `state.closest_vertices` holds a vector of all vertices in the graph ordered from closest to farthest.
 
+* `maxdist::Float64` (default: `Inf`) specifies the maximum path distance beyond which all path distances are assumed to be infinite (that is, they do not exist).
```

Comment:
Does it have to be a `Float64`? It seems like this could be replaced with `typemax(eltype(distmx))`?

Reply:
yes! I have just committed your suggestion. Thank you!